src/__tests__/checkVersion.test.ts

```diff
@@ -48,8 +49,9 @@ describe('checkVersion', function () {
   })
 
   test('assert comparisons - pass  ', async function () {
-    const setFailedStubx = sinon.stub(core, 'setFailed')
-    const checkVersion = new CheckVersion(core, fs)
+    const mock = {...core}
```

Comment:
core? is it like a mocked version of an instance? (i assuma core is github's object) but would it work with a class and have like default stubs for all methods?

Reply:
So `core` here is the imported github actions module. I have to do this weird mock creation because otherwise `sinon` complains:

```
 TypeError: ES Modules cannot be stubbed
```